packages/lesswrong/server/resolvers/revisionResolvers.ts

```diff
@@ -113,7 +113,11 @@ addFieldsDict(Revisions, {
         const mainTextHtml = sanitizeHtml(
           html, {
             allowedTags: _.without(sanitizeAllowedTags, 'blockquote', 'img'),
-            nonTextTags: ['blockquote', 'img', 'style']
+            nonTextTags: ['blockquote', 'img', 'style'],
+            
+            exclusiveFilter: function(element: IFrame) {
```

Comment:
why does this involve iframeS?

Reply:
This is a misleading type name from `sanitize-html`. The argument that `exclusiveFilter` tags can be any HTML element (in this case it's a div), but the type it receives is named IFrame, presumably because this was once an iframe-specific mechanism that was later generalized.